utils/mock-data-loader/tfm/insert-mocks-tfm.js

```diff
@@ -7,7 +7,7 @@ const insertMocks = async (token) => {
   await Promise.all(createTeams);
 
   console.info('inserting TFM users');
-  const createUsers = MOCKS.USERS.map((user) => api.createTfmUser(user, token));
+  const createUsers = MOCKS.USERS.map(async (user) => api.createTfmUser({ ...user, azureOid: user.username }, token));
```

Comment:
Maybe could use `Promise.all` here

Reply:
We have another user story where mock loading will be revamped. Not sure if it includes `users` inserting or just `deals`.

Reply:
Maybe it was improved in other PR, maybe not, but this one is already quite big.